python/marvin/contrib/vacs/firefly.py

```diff
@@ -45,10 +51,20 @@ def set_summary_file(self, release):
         # define the variables to build a unique path to your VAC file
         # look up the MaNGA drpver from the release
         drpver, dapver = config.lookUpVersions(release)
-        self.path_params = {'ver': self.version[release], 'drpver': drpver}
+		
+        self.path_params = []
+        if release=='DR17':
+            
+            self.path_params.append({'ver': self.version[release], 'drpver': drpver, 'models':'miles'})
+            self.path_params.append({'ver': self.version[release], 'drpver': drpver, 'models':'mastar'})
+			
+            self.summary_file = [self.get_path('mangaffly', path_params=self.path_params[i]) for i in [0,1]]
+			
+        else:
+            self.path_params.append({'ver': self.version[release], 'drpver': drpver})
 
-        # get_path returns False if the files do not exist locally
-        self.summary_file = self.get_path('mangaffly', path_params=self.path_params)
+            # get_path returns False if the files do not exist locally
+            self.summary_file.append(self.get_path('mangaffly', path_params=self.path_params))
```

Comment:
I don't see where `summary_file` gets defined as a list?  Should this be an `.append` if the `summary_file` is a single file for pre-DR17 releases?  Looks like the DR17 case is explicitly captured in line 61.

Reply:
Good point. Fixed it.